app/controllers/users_controller.rb

```diff
@@ -0,0 +1,35 @@
+class UsersController < ApplicationController
+  def edit
+    @user = User.find(params[:id])
```

Comment:
Ir vērts pačekot `before_action` hooks

Reply:
Did it , ieliku lai user mainigais definejas pirms actioniem